test/test_public_bindings.py

```diff
@@ -363,6 +363,7 @@ def onerror(modname):
             "torch._inductor.codegen.cpp_micro_gemm",
             "torch._inductor.codegen.cpp_template_kernel",
             "torch._inductor.runtime.triton_helpers",
+            "torch._inductor.template_heuristics.cutedsl",
```

Comment:
Any way we could make this module actually importable? It's weird to ship in our package something that looks like a submodule but should not be imported